Sources/Xcodeproj/XcodeProjectModelSerialization.swift

```diff
@@ -398,23 +398,14 @@ extension PropertyListDictionaryConvertible {
                 preconditionFailure("unnamed build settings are not supported")
             }
             switch child.value {
-              case nil:
+              case Optional<Any>.none:
```

Comment:
Perhaps `case Optional<String>.none, Optional<[String]>.none` would be more accurate?

Reply:
Swift doesn't like it.

````
swift-source/swiftpm/Sources/Xcodeproj/XcodeProjectModelSerialization.swift:401:35: error: cannot downcast from 'Any' to a more optional type 'Optional<String>'
            case Optional<String>.none, Optional<[String]>.none:
                                  ^
swift-source/swiftpm/Sources/Xcodeproj/XcodeProjectModelSerialization.swift:401:60: error: cannot downcast from 'Any' to a more optional type 'Optional<[String]>'
            case Optional<String>.none, Optional<[String]>.none:
                                                           ^
````

Reply:
Ah, right, because the type of `child.value` is actually `Optional<Any>`